docs/application/web/guides/machine-learning/single.md

```diff
@@ -119,6 +119,39 @@ To invoke the neural network model, you need to create the `TensorsData` instanc
     var result = model.invoke(inputTensorsData);
     ```
 
+## Invoke asynchronously
+
+In case when inference takes more time it is recommended to use `invokeAsync()` method. It will help not to freeze the application UI.
```

Comment:
```suggestion
In case when inference takes more time it is recommended to use `invokeAsync()` method. It preserves responsive application UI during the inference.
```